docker-compose.yaml

```
@@ -0,0 +1,18 @@
+version: "3.8"
+services:
+  api:
+    image: tiunovnn/hse-knee-api:v2
```

```suggestion
    image: cr.yandex/crp9d2qhe6l8kdja1v3i/api:latest
```
Настроена автосборка на этот container-registry. И там требуется авторизация. Без нужных ключей его не поднять